attributes/default.rb

```diff
@@ -58,6 +58,9 @@
 default['cfncluster']['nvidia']['enabled'] = 'no'
 default['cfncluster']['nvidia']['driver_url'] = 'http://download.nvidia.com/XFree86/Linux-x86_64/418.56/NVIDIA-Linux-x86_64-418.56.run'
 default['cfncluster']['nvidia']['cuda_url'] = 'https://developer.nvidia.com/compute/cuda/10.0/Prod/local_installers/cuda_10.0.130_410.48_linux'
+if node['cfncluster']['cfn_region'].start_with?("cn-")
```

Comment:
We could move the "if" closer to the driver_url initialization or maybe use an if/else block